src/blocks/transformers/temporaryInfo/temporaryPanelProtocol.ts

```diff
@@ -50,7 +71,7 @@ export async function stopWaitingForTemporaryPanels(nonces: UUID[]) {
   expectContext("contentScript");
 
   for (const nonce of nonces) {
-    panels.get(nonce)?.resolve();
+    panels.get(nonce)?.registration?.resolve();
```

Comment:
I think `registration` is always defined

```suggestion
    panels.get(nonce)?.registration.resolve();
```